src/api/api.js

```diff
@@ -0,0 +1,52 @@
+import Appwrite from "appwrite";
+import { Server } from "../utils/config";
+
+let api = {
+  sdk: null,
+
+  provider: () => {
+    if (api.sdk) return api.sdk;
```

Comment:
It's really depands on linting rule but... this is a bad practice. You might want to use lint if you want this to be an example app.

Reply:
You mean not including the braces around if?

Reply:
Yeah. The linter will catch it and the other missing curley braces.